consumer/migration/hermes_migrator.go

```diff
@@ -308,3 +327,23 @@ func (m *HermesMigrator) isRegistered(chainID int64, id string) (bool, error) {
 	}
 	return status == registry.Registered, nil
 }
+
+func (m *HermesMigrator) markAsMigrated(id string) {
+	err := m.db.SetValue(hermesMigrationBucketName, m.getMigrationKey(id), true)
```

Comment:
This is bad. Current hermesID should be part of the migration bucket key, so that we can continue migration to another hermes as it changes.

Reply:
Good point